packages/sdk/src/producers/webVitals.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  onLCP,
+  onINP,
+  onCLS,
+  onTTFB,
+  type LCPMetric,
+  type INPMetric,
+  type CLSMetric,
+  type TTFBMetric,
+} from 'web-vitals';
+import { Trackable } from '../Trackable';
+
+type Metric = LCPMetric | INPMetric | CLSMetric | TTFBMetric;
+
+type WebVitalsMetricMessage = {
+  metric_id: string;
+  metric_delta: number;
+};
+declare module '../events' {
+  export interface Event {
+    'web-vitals-lcp'?: WebVitalsMetricMessage;
+    'web-vitals-inp'?: WebVitalsMetricMessage;
+    'web-vitals-cls'?: WebVitalsMetricMessage;
+    'web-vitals-ttfb'?: WebVitalsMetricMessage;
+  }
+}
```

Comment:
Need to revisit this. The idea is to make semantic events typed, but it's not fully setup yet. Either fix completely or ditch for now.